vocabularies/Org.OData.Temporal.V1.xml

```diff
@@ -149,22 +152,27 @@
           If present, they MUST have the same type, either `Edm.Date` or `Edm.DateTimeOffset`,
           and they are interpreted according to the [`ApplicationTimeSupport/UnitOfTime`](#ApplicationTimeSupportType) of the collection.
           In particular, `ApplicationTimeSupport/UnitOfTime/ClosedClosedPeriods` governs whether a `PeriodEnd` of type `Edm.Date`
-          is the last day in the period or the first day after the period.</String>
+          is the last day in the period or the first day after the period.
+          If `PeriodStart` is present and `PeriodEnd` is absent, a default value of "ad infinitum" is assumed for `PeriodEnd`.</String>
         </Annotation>
-        <Property Name="Timeslice" Type="Edm.EntityType" Nullable="false">
-          <Annotation Term="Core.Description" String="A time slice with the same entity type as the binding parameter of the action" />
-        </Property>
         <Property Name="PeriodStart" Type="Edm.PrimitiveType" Nullable="true">
           <Annotation Term="Core.Description" String="Lower boundary of the time slice" />
         </Property>
         <Property Name="PeriodEnd" Type="Edm.PrimitiveType" Nullable="true">
           <Annotation Term="Core.Description" String="Upper boundary of the time slice" />
         </Property>
+        <NavigationProperty Name="Timeslice" Type="Edm.EntityType" Nullable="false">
+          <Annotation Term="Core.AutoExpand" />
+          <Annotation Term="Core.Description" String="A time slice with the same entity type as the binding parameter of the action" />
+          <Annotation Term="Core.LongDescription">
+            <String>The time slice has the same entity set as the binding parameter value.</String>
+          </Annotation>
```

Comment:
I would remove this sentence: the time slices in the request body are "instructions" on how to modify the entity set the action is called on, they aren't full entities.

```suggestion
```

Reply:
But after `$apply=groupby(...,aggregate(...))`, the result doesn't contain "full" entities either, and still we consider it to have the same `@odata.context` as before. I see it the same way here.